monitoring/pra/alerts.yaml

```diff
@@ -0,0 +1,79 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: kafka_connect_src_job
+    type: config
+    value: artesca-data-dr-source-base-queue-connector-metrics
+  - name: kafka_connect_sink_job
+    type: config
+    value: artesca-data-dr-base-queue-connector-metrics
+  - name: dr_sink_instance
+    type: config
+    value: artesca-data-dr
+  - name: RTO
+    type: constant
+    value:  86400 # 24 hours
```

Comment:
RTO is probably not 24 hours : we are expected to perform the whole transition in 24h, so PRA must manage metadata in much less than that... probably 1 hour is a lot already

Reply:
the 2 metrics are in millisecond, so this should be millisecond as well